test.js

```diff
@@ -68,11 +73,38 @@ test('check if different elements ready inside different targets with same selec
 	t.is(element2.id, 'unicorn2');
 });
 
+test('check if element ready after dom loaded', async t => {
+	const elementCheck = elementReady('#bio', {
+		cancelOnDomLoaded: true
+	});
+
+	delay(50000).then(() => {
```

Comment:
Use await. Like: https://github.com/sindresorhus/element-ready/commit/513139a880cc2808666ebfae6c2d47b89b624379#diff-1dd241c4cd3fd1dd89c570cee98b79ddR16

Reply:
Can't, it needs to happen after line 87 (`await elementCheck`), and we don't actually need to await it, `elementCheck` is supposed to "cancel on dom ready" and return `null` _before_ that code is run, hence the long timer.